public/app/features/query/components/QueryGroup.tsx

```diff
@@ -240,7 +241,13 @@ export class QueryGroup extends PureComponent<Props, State> {
 
   onAddQuery = (query: Partial<DataQuery>) => {
     const { dsSettings, queries } = this.state;
-    this.onQueriesChange(addQuery(queries, query, { type: dsSettings?.type, uid: dsSettings?.uid }));
+    this.onQueriesChange(
+      addQuery(queries, query, {
+        type: dsSettings?.type,
+        uid: dsSettings?.uid,
+        apiVersion: dsSettings?.meta.apiVersion,
```

Comment:
hymm -- I though we were not going to add this to the swagger yet?

I don't think we want that form locked in yet